jest.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   preset: "ts-jest",
-  // this test environment adds TextEncoder to jsdom, which we require.
```

Comment:
why delete this comment?

Reply:
Addressed in https://github.com/inrupt/solid-client-access-grants-js/pull/264/commits/a93ccd6ce956bf2a08a902bb973dd6cf99e49105